frame/executive/src/lib.rs

```diff
@@ -541,10 +609,22 @@ where
 		extrinsics: Vec<Block::Extrinsic>,
 		block_number: NumberFor<Block>,
 	) {
-		extrinsics.into_iter().for_each(|e| {
-			if let Err(e) = Self::apply_extrinsic(e) {
-				let err: &'static str = e.into();
-				panic!("{}", err)
+		sp_runtime::runtime_logger::RuntimeLogger::init();
+		extrinsics.into_iter().for_each(|tx| {
+			let tx_hash = BlakeTwo256::hash(&tx.encode());
+			let is_extrinsic = tx.is_signed().unwrap();
+			if let Err(e) = Self::apply_extrinsic(tx) {
+				log::debug!(target: "runtime::ver", "executing extrinsic :{:?}", tx_hash);
+				// there will be some cases when tx execution may fail (because of delayed execution) so we want to panic only when:
+				// - tx is inherent
+				// - tx is extrinsic and error cause is exhaust resources
+				if !is_extrinsic || matches!(e, TransactionValidityError::Invalid(err) if err.exhausted_resources())
```

Comment:
we allow some txs from queue to fail in some corner cases that we may not be aware